index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Technical Documentation</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+
+  <nav id="navbar">
+    <header>What is HTML</header>
+    <a class="nav-link" href="#Introduction_to_html"> Introduction to html</a>
+    <a class="nav-link" href="#Basic_structure_of_html">Basic structure of html</a>
+    <a class="nav-link" href="#Commen_html_elements"> Commen html elements</a>
+    <a class="nav-link" href="#Attribute_in_html"> Attribute in html</a>
+    <a class="nav-link" href="#Html_style"> Html style</a>
+  </nav>
+
+  <main id="main-doc">
+    <section id="Introduction_to_html" class="main-section">
+        <header> Introduction to html</header>
+        <p>HTML stands for Hyper Text Markup Language and it is the standard markup language for creating Web pages. HTML consists of a series of elements, the elements tell the browser how to display the content.</p>
+        <p>This is the example of html elements</p>
+        <code>&lt;html&gt;</code>
+        <code>&lt;body&gt;</code>
+        <h4>The important elements in html is:</h4>
+        <ul>
+            <li> html </li>
+            <li> head </li>
+            <li> body</li> 
+        </ul>
```

Comment:
@MasomRezaie 

Well done you implemented the feedback very well, but still in here, you used 4 spaces instead of 2 spaces try to use the 2 spaces indentation..